diff --git a/node_modules/react-native-video/lib/Video.js b/node_modules/react-native-video/lib/Video.js
index 2dbeed3..d47c8e6 100644
--- a/node_modules/react-native-video/lib/Video.js
+++ b/node_modules/react-native-video/lib/Video.js
@@ -107,7 +107,7 @@ const Video = (0, react_1.forwardRef)(({ source, style, resizeMode, poster, post
         }
         const isNetwork = !!(uri && uri.match(/^(rtp|rtsp|http|https):/));
         const isAsset = !!(uri &&
-            uri.match(/^(assets-library|ipod-library|file|content|ms-appx|ms-appdata|asset):/));
+            uri.match(/^(assets-library|ipod-library|file|content|ms-appx|ms-appdata|asset|data):/));
         const selectedDrm = _source.drm || drm;
         const _textTracks = _source.textTracks || textTracks;
         const _drm = !selectedDrm
diff --git a/node_modules/react-native-video/src/Video.tsx b/node_modules/react-native-video/src/Video.tsx
index 789d915..174783c 100644
--- a/node_modules/react-native-video/src/Video.tsx
+++ b/node_modules/react-native-video/src/Video.tsx
@@ -223,7 +223,7 @@ const Video = forwardRef<VideoRef, ReactVideoProps>(
         const isAsset = !!(
           uri &&
           uri.match(
-            /^(assets-library|ipod-library|file|content|ms-appx|ms-appdata|asset):/,
+            /^(assets-library|ipod-library|file|content|ms-appx|ms-appdata|asset|data):/,
           )
         );
 
