diff --git a/node_modules/react-native-quick-base64/react-native-quick-base64.podspec b/node_modules/react-native-quick-base64/react-native-quick-base64.podspec
index eb43697..9c5f8a3 100644
--- a/node_modules/react-native-quick-base64/react-native-quick-base64.podspec
+++ b/node_modules/react-native-quick-base64/react-native-quick-base64.podspec
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.license      = package["license"]
   s.authors      = package["author"]
 
-  s.platforms    = { :ios => min_ios_version_supported }
+  s.platforms    = { :ios => min_ios_version_supported, :tvos => "16.4" }
   s.source       = { :git => "https://github.com/craftzdog/react-native-quick-base64.git", :tag => "#{s.version}" }
 
   s.source_files = "ios/**/*.{h,m,mm}", "cpp/**/*.{hpp,cpp,c,h}", "ios/generated/*.{h,cpp,mm}"
